recipe/meta.yaml

```diff
@@ -1,12 +1,14 @@
-{% set version = "7.83.0" %}
+{% set version = "7.84.0" %}
 
 package:
   name: curl_split_recipe
   version: {{ version }}
 
 source:
   url: https://curl.haxx.se/download/curl-{{ version }}.tar.bz2
-  sha256: 247c7ec7521c4258e65634e529270d214fe32969971cccb72845e7aa46831f96
+  sha256: 702fb26e73190a3bd77071aa146f507b9817cc4dfce218d2ab87f00cd3bc059d
+  patches:
+    - sched_s390x.patch  # [s390x or ppc64le]
```

Comment:
I would rename this patch to something like `sched_yield.patch` since it's not s390x specific.

In fact, if possible, I would drop the selector entirely, and see if the `#ifdef` in the patch itself works on all our target platforms.

Reply:
I renamed the patch to `sched_yield.patch` and removed the selectors and was able to build it again.